hack/functests.sh

```diff
@@ -45,6 +45,9 @@ function functest() {
         echo "Will skip test asserting the cluster is in dual-stack mode."
         extra_args="${extra_args} -skip-dual-stack-test"
     fi
+
+    extra_args="${extra_args}  -conn-check-ipv4-address=${conn_check_ipv4_address} -conn-check-ipv6-address=${conn_check_ipv6_address} -conn-check-dns=${conn_check_dns}"
```

Comment:
maybe if the conn-check-ipvX-address are empty lets not add them to the command ?
same for the dns

Reply:
Done.

Reply:
double space before `-conn-check-ipv4-address`